bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -204,7 +203,18 @@ default Border border(int col)     {
 
         static final class GroupRow extends AbstractRow {
             private static String extractName(final GuideGroup group) {
-                return group.getName().getOrElse("Auto");
+                final GuideGrp grp = group.grp();
+                final String name;
+                if (grp.isAutomatic()) {
+                    if (grp instanceof AutomaticGroup.Disabled$) {
+                        name = "Auto (Disabled)";
```

Comment:
This is currently done in `EdCompTargetList.setSelectionFromNode`.

The code for this should probably be moved directly into `EdCompTargetList.updateUIForGroup`, but that is a change for another PR.